DOCUMENTATION.md

```diff
@@ -37,61 +38,114 @@ The output tensor `ys` will have shape `(t_size, batch_size, state_size)`, and c
 - `diagonal`: The diffusion `g` is element-wise and has output size `(batch_size, state_size)`. The Brownian motion is a batch of `state_size`-dimensional Brownian motions.
 - `general`: The diffusion `g` has output size `(batch_size, state_size, brownian_size)`. The Brownian motion is a batch of `brownian_size`-dimensional Brownian motions.
 
+Whilst `scalar`, `additive` and `general` are special cases of `general` noise, they allows for additional solvers to be used and additional optimisations to take place.
```

Comment:
`scalar`, `additive`, and `diagonal`? 

"they allows for ..." -> "they allow for"